crates/gpui/src/platform/mac/window.rs

```diff
@@ -1827,6 +1832,10 @@ extern "C" fn accepts_first_mouse(this: &Object, _: Sel, _: id) -> BOOL {
     YES
 }
 
+extern "C" fn character_index_for_point(_this: &Object, _: Sel, _position: NSPoint) -> u64 {
+    NSNotFound as u64
```

Comment:
I'm not sure if we want this solution, but not crashing is > crashing. So lets at least mark this with a TODO :)

Reply:
100% agree, I am just unsure how to do that correctly.

Not sure on the guidelines on TODOs, but I added an explanation.